themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/my-resources-page/PermissionRequest.tsx

```diff
@@ -59,7 +59,7 @@ export class PermissionRequest extends React.Component<PermissionRequestProps, P
         const permissions: Permission[] = permissionsRequest.data || [];
 
         // Erik - I had to add the exclamation point.  Can we be sure that the 'find' will not return undefined?
```

Comment:
Do we know the answer to my question here?

Reply:
yep it's not always there I fixed that in #7067